react-native-adjust.podspec

```diff
@@ -0,0 +1,22 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name           = 'react-native-adjust'
+  s.version        = package['version']
+  s.summary        = package['description']
+  s.description    = package['description']
+  s.license        = package['license']
+  s.author         = package['author']
+  s.homepage       = package['homepage']
+  s.source         = { :git => 'https://github.com/adjust/react_native_sdk', :tag => s.version }
+
+  s.requires_arc   = true
+  s.platform       = :ios, '8.0'
+
+  s.preserve_paths = 'LICENSE', 'README.md', 'package.json', 'index.js'
+  s.source_files   = 'ios/*.{h,m}'
```

Comment:
Ah, we were using this repo directly without adding the Adjust iOS SDK pod in our Profile. This issue is up to Adjust on how they want to manage the project. Given that the iOS SDK is currently embedded in this project, the podspec would be more congruent if it didn't have a separate dependency and used the recursive glob. The recommended way to embed this project into an iOS app is with `react-native link`, and I think for that, having the iOS extension as part of the project is a best practice. Either approach would be great so that we can stop using our fork.